src/tooling/piral-cli/src/apps/new-pilet.test.ts

```diff
@@ -40,6 +41,8 @@ describe('New Pilet Command', () => {
   });
 
   it('command will fail when providing invalid registry', async () => {
+    jest.setTimeout(100000);
```

Comment:
I think this is nearly 2 minutes, right? Is 60000 enough? Is shorter maybe even better?

Reply:
Successfully building a pilet (even in my machine with the latest Core i7 processor) was exceeding 60 seconds.  I think I also have sufficient network bandwidth for that to not be the issue.  I suspect there is more processing overhead with the new NPM version.

We want the timeout to be just above a common length of time for the test to succeed.  This particular test was clocking something in the range of 70+ seconds for me.

Reply:
Well, this one should not take so long at all (I guess you use it on Windows?) because we have an invalid registry.

In any case we can also remove this test altogether. The cli integration tests cover that anyway pretty well and in here we should rely mostly on unit tests / i.e., mock away such things.